### DISCUSSION TITLE
Skip folders beginning with a dot?

### DISCUSSION THREAD

**Discussion by jftuga** [tavianator/bfs#114] [General]
How do I skip all folders that start with a `.`?  How would I do the same, but not at just the top-level?

Thanks.

**Comment by jftuga** [comment]
Just found the answer: `-nohidden`

**Reply by tavianator** [reply]
If you only want to skip hidden *directories*, not files, you could do it like this:

```console
$ bfs -exclude \( -type d -hidden \)
```
